ExpenseTracker/Domain/Expense.cs

```diff
@@ -19,6 +21,10 @@ public Expense(Guid userId, decimal amount, string description, string category,
         CreatedAt = DateTime.UtcNow;
     }
 
+    public Expense()
```

Comment:
Constructor vacio

Reply:
just as an example, since we are not taking into account the delete endpoint, I put it as a base in case we use it in the future.